src/main/java/nextstep/step2/domain/Line.java

```diff
@@ -0,0 +1,23 @@
+package nextstep.step2.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Line {
+    private List<Point> points = new ArrayList<>();
+
+    private Line(final int countOfParticipant){
```

Comment:
```suggestion
    private Line(final int countOfParticipant) {
```

단축키를 사용하여 정렬하는 방법을 찾아 보시면 좋을 것 같아요.